### PR TITLE
ktop plugin

### PR DESCRIPTION
**Pull request by vladimirvivien**
This patch introduces ktop as a kubectl plugin.

### Local installation

```
kubectl krew install --manifest dist/ktop.yaml
Installing plugin: ktop
Installed plugin: ktop
\
 | Use this plugin:
 | 	kubectl ktop
 | Documentation:
 | 	https://github.com/vladimirvivien/ktop
 | Caveats:
 | \
 |  | * By default, ktop displays metrics for resources in the default namespace. You can override this behavior
 |  | by providing a --namespace or use -A for all namespaces.
 | /
/
```

### Local verification

```
kubectl ktop
Connected to: https://192.168.64.21:8443

 _    _
| | _| |_ ___  _ __
| |/ / __/ _ \| '_ \
|   <| || (_) | |_) |
|_|\_\\__\___/| .__/
              |_|
Version v0.2.0 (426916f7d1eb3d9da0586970b112f21d6ebdfff0)
```

### Plugin help

```
 kubectl ktop --help
Runs kubectl-ktop as kubectl plugin

Usage:
  kubectl-ktop [flags]

Examples:

# Start ktop using default configuration for the "default" namespace
kubectl-ktop

# Start ktop with default configuration for all accessible namespaces
kubectl-ktop -A

# Start ktop for a specific namespace in current context
kubectl-ktop --namespace <namespace>

# Start ktop for a specific namespace and context
kubectl-ktop --namespace <namespace> --context <context>


Flags:
  -A, --all-namespaces                 If true, display metrics for all accessible namespaces
      --as string                      Username to impersonate for the operation. User could be a regular user or a service account in a namespace.
      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
      --as-uid string                  UID to impersonate for the operation.
      --cache-dir string               Default cache directory (default "/Users/vivienv/.kube/cache")
      --certificate-authority string   Path to a cert file for the certificate authority
      --client-certificate string      Path to a client certificate file for TLS
      --client-key string              Path to a client key file for TLS
      --cluster string                 The name of the kubeconfig cluster to use
      --context string                 The name of the kubeconfig context to use
  -h, --help                           help for kubectl-ktop
      --insecure-skip-tls-verify       If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
      --kubeconfig string              Path to the kubeconfig file to use for CLI requests.
  -n, --namespace string               If present, the namespace scope for this CLI request
      --request-timeout string         The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests. (default "0")
  -s, --server string                  The address and port of the Kubernetes API server
      --tls-server-name string         Server name to use for server certificate validation. If it is not provided, the hostname used to contact the server is used
      --token string                   Bearer token for authentication to the API server
      --user string                    The name of the kubeconfig user to use
```

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

File: plugins/ktop.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          os: darwin
+          arch: arm64
+    - bin: kubectl-ktop
+      uri: https://github.com/vladimirvivien/ktop/releases/download/v0.2.0/kubectl-ktop_v0.2.0_darwin_amd64.tar.gz
+      sha256: a4348ea29fc11821292e78b7f114cc5a9d441fa0404b5b7f1c65fcf57a50f1c8
+      selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+  shortDescription: A top tool for kubernetes to display workload metrics
```
Comment (ahmetb) [inline plugins/ktop.yaml:36]: "for Kubernetes" is redunant (strive for 50 chars max for this field)

**Review by ahmetb — COMMENTED** [review]
/approve

File: plugins/ktop.yaml (hunk 1 above)
Comment (vladimirvivien) [reply plugins/ktop.yaml]: Fixed! Thanks.

**Comment by vladimirvivien** [comment]
@ahmetb please take a look.

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!


Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**